packages/eslint-config/index.js

```diff
@@ -79,6 +79,7 @@ module.exports = {
           'error',
           { ignoreFunctionalComponents: true },
         ],
+        'import/no-extraneous-dependencies': 'off',
```

Comment:
What was this causing a lint error on? I don't think we should turn this off for all TS files

If it's the `jest.setup.ts` then just add a 1 time ignore there. Test files ignore this error because test util packages should be dev dependencies since they aren't required for prod.

Reply:
It was actually the unused `embed-grid/setupTests.ts` file, I've just deleted that and added this rule back.

That being said - a few lines above the rule is already applied to all js/jsx/ts/tsx files - if I add that rule back, there's about 30 failures in test files. That can be saved for another ticket.

Reply:
That rule is only applied to `test.{js,jsx,ts,tsx}` files which I think is fine. It triggers test failures because we import dev dependencies for tests sometimes. But those dependencies should be left as dev since they aren't used in prod code

Reply:
Ah. My dumb ass just can't read a simple regex. My bad! Good call.